src/SFA.DAS.LevyTransferMatching.Web/Models/Opportunities/ApplicationDetailsPostRequest.cs

```diff
@@ -26,5 +26,6 @@ public class ApplicationDetailsPostRequest : ApplyRequest
         public bool? HasTrainingProvider { get; set; }
         public string SelectedStandardId { get; set; }
         public string SelectedStandardTitle { get; set; }
+        public double Amount { get; set; }
```

Comment:
Unless I've misunderstood, this property looks like it's just been tacked on to have somewhere to put the calculated value later, and it's not really a part of the posted data, in which case it'd be worth removing from here (see later comments).